packages/dashboards/addon/components/dashboard-action-list.hbs

```diff
@@ -48,7 +48,17 @@
             @onSave={{delivery-rule-action "SAVE_DELIVERY_RULE"}}
             @onRevert={{delivery-rule-action "REVERT_DELIVERY_RULE"}}
             @onDelete={{delivery-rule-action "DELETE_DELIVERY_RULE"}}
+            as |onOpen|
           >
+            <button 
+              type="button"
+              id="navi-dashboard-action-schedule-{{@index}}"
+              disabled={{not dashboard.validations.isTruelyValid}}
+              class="schedule schedule-action__button"
+              {{on "click" onOpen}}
+            >
+              <NaviIcon @icon="clock-o" class="schedule" />
```

Comment:
Can we clean up the use of the `schedule` class. In this file/navi-action-list/report-actions. It looks like we use sometimes use it
- on the modal itself
- on the button to open the modal
- on the icon inside the button